### PR TITLE
Add keukenhof

### PR DESCRIPTION
**Pull request by Alexandrshy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -762,6 +762,7 @@ Besides libraries, there're [Collection on Codepen](http://codepen.io/collection
 * [fancyBox](https://github.com/fancyapps/fancyBox) - A tool that offers a nice and elegant way to add zooming functionality for images, html content and multi-media on your webpages.
 * [swipebox](https://github.com/brutaldesign/swipebox) - A touchable jQuery lightbox
 * [jBox](https://github.com/StephanWagner/jBox) - jBox is a powerful and flexible jQuery plugin, taking care of all your popup windows, tooltips, notices and more.
+* [keukenhof](https://github.com/Alexandrshy/keukenhof) - Lightweight, no dependencies, accessibility enabled TypeScript library for creating modal windows.
```
Comment (gokaygurcan) [inline README.md:765]: Don't you think it's _too early_ to consider this one as _awesome_?

Comment (Alexandrshy) [reply README.md:765]: It's up to you, I just don't see in the description that the proposed libraries should have a certain number of stars or something like that

Comment (lofcz) [reply README.md:765]: The lib works, it would be fair to _consider it_.

Comment (gokaygurcan) [reply README.md:765]: okay guys, you convinced me. that's a bit off to check in a repo with 2 stars and no followers but i agree it's doing it's job and it's being maintained. maybe this way, it'll get the popularity and awesomeness that it deserves. 

please fix the conflicts and update the pr, then let's get in.

Comment (Alexandrshy) [reply README.md:774]: Thanks guys, done

**Review by gokaygurcan — APPROVED** [review]
let's give it chance to shine.
thanks!

**Pull request merged**